README.md

```diff
@@ -288,9 +288,12 @@ ${hostedzone://example.com/private}
 
 Looks up an AMI given a name prefix which may contain wildcards. If query returns multiple images the latest is used.
 
+Owners takes a query string list of AWS account ID, self (owner is the sender of the request), or an AWS owner alias (valid values are amazon | aws-marketplace | microsoft). Omitting this option returns all images for which you have launch permissions, regardless of ownership.
```

Comment:
The documentation here looks like it doesn't match the code. If owner is omitted, doesn't it default to "self"?

I think defaulting to "self" is probably the behaviour we want, as it's backwards compatible and less likelihood of ending up with the wrong image.